charts/multicluster-controlplane/values.yaml

```diff
@@ -0,0 +1,14 @@
+# Default values for multicluster-controlplane.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: quay.io/open-cluster-management/multicluster-controlplane:latest
+imagePullPolicy: IfNotPresent
+
+replicas: 1
+
+configfile: "ocmconfig.yaml"
```

Comment:
TODO for config, we may need another pr to handle, I don't think we should put a ocmconfig.yaml here, we may need special configurations here

Reply:
```
apiserver:
  externalHostname: ""
  port: 443
  caFile: ""
  caKeyFile: ""

etcd:
  mode: embed
  prefix: "/registry"
  servers: []
  caFile: ""
  certFile: ""
  keyFile: ""
```

Reply:
in the secret, we may need

```
apiVersion: v1
kind: Secret
metadata:
  name: controlplane-config
  namespace: {{ .Release.Namespace }}
type: Opaque
data:
  ocmconfig.yaml: |-
           apiserver:
               extrnalHostName: "{{ .apiserver.externalHostname}}"
               port:
           etcd:
              ...
   <if apiserver ca file is not empty>
   apiserver-ca.crt:
        ....
  <if etcd != embed>
etch-ca.crt:
...
```

Reply:
or 

```
apiVersion: v1
kind: Secret
metadata:
  name: controlplane-config
  namespace: {{ .Release.Namespace }}
type: Opaque
data:

           apiserverextrnalHostName: "{{ .apiserver.externalHostname}}"
            apiserverextrnalport:
           etcdMode:
   <if apiserver ca file is not empty>
   apiserver-ca.crt:
        ....
  <if etcd != embed>
etch-ca.crt:
...
```